lms/djangoapps/verify_student/services.py

```diff
@@ -92,6 +93,11 @@ def get_verified_user_ids(cls, users):
             'created_at__gt': now() - timedelta(days=settings.VERIFY_STUDENT["DAYS_GOOD_FOR"])
         }
         return chain(
+            VerificationAttempt.objects.filter(**{
+                'user__in': users,
+                'status': 'approved',
+                'created__gt': now() - timedelta(days=settings.VERIFY_STUDENT["DAYS_GOOD_FOR"])
```

Comment:
Ah, is this because `TimestampedModel` changed the name of `created_at` to `created` at some point? 🤦‍♂️

Reply:
It's actually not even a timestamped model. There are explicit fields for created_at and updated_at instead of created/modified. Unfortunately I can't just use a model property here since this is a query.

Reply:
Ah, I see. I'm tempted to just name the fields `created_at` and `updated_at` in the generic model, but I think using `TimestampedModel` is a better approach longer term.